network/hostnetwork/hostnetwork_test.go

```diff
@@ -140,6 +140,21 @@ func mockConfiguration(address string) configuration.Configuration {
 	return result
 }
 
+func assertTimeout(t *testing.T, wg *sync.WaitGroup) {
```

Comment:
I removed a function like this in previous pull request :)
Call `wg.Wait()` instead of this function and let the go test toolkit control the tests timeouts